style.css

```diff
@@ -0,0 +1,104 @@
+.ticket-container {
+    margin-top: 50px;
+    margin-bottom: 50px;
+    width: 782px;
+    height: 273px;
+    background: #EFC684;
+    border-radius: 10px;
+    display: flex;
+    font-family: Roboto;
+}
+
+.ticket-main {
+    width: 289px;
+}
+
+.ticket-text {
+    width: 206px;
+    height: 128px;
+    font-size: 55px;
+    margin-top: 26px;
+    margin-left: 20px;
+}
+
+.ticket-details {
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 63px;
```

Comment:
Using such a big margin value is always a red flag. Probably you can achieve the same effect by making the parent component bigger and setting up the flexbox align/justify-content.